dataprofiler/profilers/float_column_profile.py

```diff
@@ -71,6 +124,12 @@ def profile(self):
         if self.histogram_selection is not None:
             histogram_method = self.histogram_selection
 
+        protected_precision = {}
+        for key in ['min', 'max']:
+            protected_precision[key] = None
+        if self.precision[key] is not None:
+            protected_precision[key] = int(self.precision[key])
```

Comment:
This needs to be nested, right?

Reply:
Also what exactly is the purpose of this dictionary?

Reply:
this should cause tests to fail, but the tests aren't actually checking this. I referenced below